spark/pom.xml

```diff
@@ -668,6 +668,16 @@
     </profile>
 
     <profile>
+      <id>hadoop-2.7</id>
```

Comment:
I see that Hortonworks HDP 2.3 released recently (or soon) has Hadoop 2.7.1, so maybe there is a reason to have that profile.


With so many profiles I can see it is increasing difficult to cover all of it in each release. Do we have any plan to trim the older ones?



On Sat, Jul 25, 2015 at 3:19 PM -0700, "Randy Gelhausen" <notifications@github.com> wrote:
> @@ -668,6 +668,16 @@
>      </profile>
>
>      <profile>
> +      <id>hadoop-2.7</id>

I have no opinion on whether a new Hadoop profile is NEEDED. I followed the trend of new profile per Hadoop release. I'm fine using -Phadoop-2.6, but then we should add a line in the README indicating Hadoop 2.7 users should continue using the -Phadoop-2.6 build option.

---
Reply to this email directly or view it on GitHub:
https://github.com/apache/incubator-zeppelin/pull/166/files#r35487366